src/domain/chains/entities/__tests__/block-explorer-uri-template.factory.ts

```diff
@@ -0,0 +1,14 @@
+import { BlockExplorerUriTemplate } from '../block-explorer-uri-template.entity';
+import { faker } from '@faker-js/faker';
+
+export default function (
+  address?: string,
+  txHash?: string,
+  api?: string,
+): BlockExplorerUriTemplate {
+  return <BlockExplorerUriTemplate>{
+    address: address ?? faker.internet.url(),
+    txHash: txHash ?? faker.internet.url(),
+    api: api ?? faker.internet.url(),
```

Comment:
While I agree that this can be used for documentation purposes, there is no logic in the CGW to validate how the url is formed so in a testing scenario, any `url` should be valid.

If we add validations to the format of the URL then this needs to be change to generate URLs in that format too.